### ISSUE TITLE
Feature Request: "Total" data in the miners table

### ISSUE THREAD

**Issue by ipr0310** [signum-network/signum-pool#48]
It would be a + to have the total of:

- Pending Balance
- Physical Capacity
- Effective Capacity
- Committed Balance,
- Shared Capacity

Maybe put the row at the bottom, or at the top of the miners table

**Comment by pir8radio** [comment]
Yes top of the miners table, a row that looks a little different. so you can see the "Pool" totals.  mock example below. 
![image](https://user-images.githubusercontent.com/16748696/129652959-bd071009-8e82-4a73-8e3e-ad7f761d2ecc.png)

**Comment by Alfa-Signum** [comment]
Good day.
I would like to suggest moving the comma up! That is, separate the thousandths with an opostrophe. It will be read much more conveniently. Look readable. Clearly. It will be more accurate in terms of mathematics.

Thanks for your hard work !!! monsieur and madam))

**Comment by Alfa-Signum** [comment]
Maybe you say me how do it, and i try do))))

**Comment by ipr0310** [comment]
Some progress is done:
![image](https://user-images.githubusercontent.com/35440957/166123985-7751482c-9742-409b-9d1e-199f973a4c77.png)

The following values from API endpoint are missing:
- Pending balance
- Committed balance
